environment.yml

```diff
@@ -0,0 +1,14 @@
+name: mtenn-env
+channels:
+  - conda-forge
+  - defaults
+  - pyg
+dependencies:
+  - h5py
+  - numpy
+  - pytorch
+  - pytorch-cluster
+  - pytorch-scatter
+  - pytorch-sparse
+  - pyg
+  - e3nn
```

Comment:
Not sure if there's any specifics on the formatting of this file that it can't have a newline at the end so I'm not going to add it myself, but if not let's add it